infogami/utils/app.py

```diff
@@ -7,8 +7,7 @@
 
 import web
 
-import flash
-import delegate as infogami_delegate
```

Comment:
This is actually used further down and should not have been removed. https://github.com/infogami/infogami/issues/4 needs to be solved more directly.

Reply:
__import delegate as infogami_delegate  # noqa__